pages/cli.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Markdown from 'react-markdown';
+import Nav from '../components/Nav';
+import Layout from './_Layout';
+import S3 from '../lib/s3';
+import locals from '../locals';
+import Table from '../components/Table';
+
+const fetchData = async () => {
+  const data = await Promise.all([
+    fetch('https://api.github.com/repos/resin-io/etcher/contents/docs/CLI-INSTALLATION.md').then(res => ( res.json() )),
```

Comment:
It gets run at build time and then again on page load. The page load checks if there are any updates, if there are updates it re-renders that component. Make sense?